observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -5,3 +5,46 @@ include::_attributes/common-attributes.adoc[]
 :context: logging-6x
 
 toc::[]
+
+The `ClusterLogForwarder` custom resource is the central configuration point for OpenShift Logging 6.0. It defines the desired state of your logging system, including inputs, outputs, and pipelines. The Cluster Logging Operator watches for changes to this resource and automatically applies the necessary configurations to the underlying logging components.
+
+== Service account
+This release of Cluster Logging requires administrators to explicitly grant log collection permissions to the service account associated with *ClusterLogForwarder*. This was not required in previous releases for the legacy logging scenario consisting of a *ClusterLogging* and, optionally, a *ClusterLogForwarder.logging.openshift.io* resource.
+
+To use the existing service account (that is, `logcollector`) from a previous release, create the following *ClusterRoleBinding*:
+
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-application-logs -z openshift-logging:logcollector
+$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z openshift-logging:logcollector
+----
+
+Additionally, create the following *ClusterRoleBinding* if collecting audit logs:
+
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-user collect-audit-logs -z openshift-logging:logcollector
```

Comment:
oc adm policy add-cluster-role-to-user collect-audit-logs system:serviceaccount:openshift-logging:logcollector